org.godotengine.godot.BaseApp.yaml

```diff
@@ -0,0 +1,76 @@
+app-id: org.godotengine.godot.BaseApp
+branch: '3.1'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: godot-runner
+
+build-options:
+  arch:
+    x86_64:
+      env:
+        # Only enable link-time optimization when targeting x86_64
+        # (causes issues on other architectures)
+        SCONS_FLAGS_EXTRA: use_lto=yes
+
+  env:
+    # Will be appended to the version string displayed in the command-line help
+    BUILD_NAME: flathub
+
+    # SCons flags common to all builds
+    # Shouldn't be quoted when used as it's a single string, not an array
+    SCONS_FLAGS: >
+      platform=x11
+      CCFLAGS=-I/app/include
+      prefix=/app
+      unix_global_settings_path=/app
+      progress=no
+      builtin_freetype=no
+      builtin_libogg=no
+      builtin_libpng=no
+      builtin_libtheora=no
+      builtin_libvorbis=no
+      builtin_libwebp=no
+      builtin_openssl=no
```

Comment:
Godot 3.1+ uses mbedTLS (scons flag is `builtin_mbedtls`) instead of OpenSSL.

Reply:
@bruvzg Good catch, I forgot to remove it after upgrading the Flatpak to 3.1.